### DISCUSSION TITLE
Migrating from Windows to docker/linux

### DISCUSSION THREAD

**Discussion by cainef** [datalust/seq-tickets#1920] [Help]
Hi,
We are looking at migrating our windows Seq installations to docker/linux.
I wasn't able to find anything specific in the documentation regarding a migration of this sort.

At the moment I am planning on deploying a vanilla Seq docker instance, doing a restore from backup, then copying the events across manually.

Will this work? is there anything else I should be considering (file paths etc )

**Comment by nblumhardt** [comment]
Hi! A backup/restore is the way to go - but it's important in this particular case for both the original and target instances to be running the same (ideally very latest) Seq version first. The backup on the original machine should be created by the same version you're restoring to.

Copying the `Stream/` folder manually will work, but you need to ensure that both the original and target Seq instances are fully stopped, for this to work.

Some Seq apps - e.g. those that integrate with SQL server, or anything with platform-specific dependencies, could pose a problem. I don't know exhaustively which apps cause trouble, but if you let me know what you're running I can take a quick look.

Hope this helps,
Nick

**Comment by cainef** [comment]
Thank you.
I have managed to migrate to a test instance using the backup/restore process, however I was unable to stop the source instance prior to copying the stream files (including metadata and flare).
When I started up the new instance all of the stream files were given the .junk extension.
Would this be due to not stopping the source instance before copying the files, or could there be another reason?

**Reply by nblumhardt** [reply]
This would be expected if the source is not stopped 👍

**Comment by cainef** [comment]
Great! I have now tested it with the shutdown on source first and it is working.
Appreciate your responses on this.
Thanks!
